kind/cmd/kind/delete/BUILD.bazel

```diff
@@ -3,7 +3,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 go_library(
     name = "go_default_library",
     srcs = ["delete.go"],
-    importpath = "k8s.io/test-infra/kind/cmd/kind/cmd/delete",
+    importpath = "k8s.io/test-infra/kind/cmd/kind/delete",
```

Comment:
Do we need to keep the second 'kind' still? `k8s.io/test-infra/kind/cmd/kind/delete` -> `k8s.io/test-infra/kind/cmd/delete`?

Reply:
Possibly not? This seemed like we could still have another CLI tool cmd/foo later. Not sure if we want or need that.

Reply:
I refactored this so `kind/cmd/kind` implements the `kind` portion of the cobra commands (and also `Run()` / `Main()`), so the structure is `kind/cmd/$CLI_COMMAND/$CLI_SUBCOMMAND/...`